### ISSUE TITLE
List of MPs ordered by % attendence at votes in this Parliament

### ISSUE THREAD

**Issue by RichardTaylor** [mysociety/theyworkforyou#612]
I personally had a call from a journalist today seeking an list of MPs ordered by attendance at votes.

I recall this is quite often requested (albeit the more common question is about MPs' attendence at Parliament which isn't recorded).

This is information we have; but don't tabulate / make sortable / show top ten (bottom ten) lists etc.

**Comment by RichardTaylor** [comment]
Currently 
http://www.publicwhip.org.uk/mps.php?sort=attendance
and
http://www.publicwhip.org.uk/mps.php?house=lords&sort=attendance

**Comment by dracos** [comment]
Yes, but it is sorted by attendance. I don't believe it was showing attendance rates at my last comment, nor from the code does it look like it has since at least 2008, comment by Francis: "Disabled 2008-10-01 by FAI, as people care about attendance when it is meaningless."
